doc/classes/ProjectSettings.xml

```diff
@@ -2909,7 +2909,7 @@
 		</member>
 		<member name="rendering/textures/vram_compression/compress_with_gpu" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], the texture importer will utilize the GPU for compressing textures, improving the import time of large images.
-			[b]Note:[/b] This only functions on a device which supports either Vulkan, D3D12, or Metal available as a rendering backend.
+			[b]Note:[/b] This only functions on a device which supports either Vulkan, D3D12, or Metal available as a rendering driver.
```

Comment:
While at it, "supports ... available" sounds weird.
```suggestion
			[b]Note:[/b] This only functions on a device which supports either Vulkan, D3D12, or Metal as a rendering driver.
```

Reply:
Also changed "D3D12" to "Direct3D 12", since it's almost always spelled out.